src/instrumentation.py

```diff
@@ -0,0 +1,124 @@
+from prometheus_client import CollectorRegistry, Counter, Gauge
+
+from src.constants import TASK_EVENT_LABELS, WORKER_EVENT_LABELS, LabelName
+
+registry = CollectorRegistry(auto_describe=True)
```

Comment:
Same as with celery projects where the `celery` object is instantiated once and reused everywhere we can do same with the registry which we need as a `singleton`